site/src/pages/GroupsPage/GroupPage.tsx

```diff
@@ -186,7 +153,9 @@ export const GroupPage: React.FC = () => {
 
                   <TableBody>
                     <ChooseOne>
-                      <Cond condition={Boolean(group?.members.length === 0)}>
+                      <Cond
+                        condition={Boolean(groupData?.members.length === 0)}
```

Comment:
Is there a need to do the boolean cast? The `===` comparison against `0` should always produce a boolean value, even if `groupData?.members.length` short-circuits to `undefined`